.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Python unit tests
+
+on: [ push ]
```

Comment:
Model this file off of https://github.com/NYPL/location-visits-poller/blob/main/.github/workflows/run-tests.yml:

Add a changelog check and prefer `on:
  pull_request:
    types: [ labeled, unlabeled, opened, reopened, synchronize ]` and